build.gradle.kts

```diff
@@ -97,6 +97,10 @@ buildTypes {
         tasks("performance:distributedPerformanceTest")
     }
 
+    create("distributedSlowPerformanceTests") {
+        tasks("performance:distributedSlowPerformanceTest")
+    }
+
     create("distributedPerformanceExperiments") {
         tasks("performance:distributedPerformanceExperiment")
```

Comment:
This is not used any more I guess?

Reply:
I was going to leave the experiment tasks in, given that I thought about creating a configuration which can be run on demand with failure checks enabled.

Hmm, maybe we should remove the experiments from the historical performance tests, given that they don't use the baselines anyway, and move then to a separate job which has the checks enabled and runs once a week as well. WDYT?

Reply:
What do you mean by "which can be run on demand"? This configuration `distributedPerformanceExperiments` can't be invoked at all from TeamCity,

Reply:
I would keep them in historical performance tests, maybe we need to change the name to "full performance tests", same as the configuration name?